core/txpool/celo_rates.go

```diff
@@ -23,11 +29,16 @@ func IsWhitelisted(exchangeRates common.ExchangeRates, feeCurrency *common.Addre
 	return ok
 }
 
+func TranslateValue(exchangeRates common.ExchangeRates, val1 *big.Int, currencyFrom *common.Address, currencyTo *common.Address) *big.Int {
```

Comment:
We have that implemented already: https://github.com/celo-org/op-geth/blob/715f16e2276b3e284b7b642e5ddfc9e39f7ce119/contracts/fee_currencies.go#L50

Reply:
awesome, though the fn I meant is translating from one custom currency to another one, not passing through gold currency